lblod/frontend-worship-decisions
# Sorting issue that makes submissions not showing

# Description
DL-5141

This PR is about temporary fixing the issue where submissions are not shown properly in the table.

Default sorting was set to `organization.naam` for now until a fix is released for `mu-cl-resource`.

## Behavior

When loging as CKB Dilsen-Stokkem no submissions are shown but they're should appear in the table. Clicking on the sorting icon from all headers except `Periode zitting / besluit` will make the submissions re-appear until no sorting is set.

# Type of change


# Related services

- [mu-cl-resource](https://github.com/mu-semtech/mu-cl-resources)

# How to test

1. Checkout on the branch `fix/switch-default-sorting`
2. `ember s --proxy https://controle.databankerediensten.abb-croco.s.redhost.be`
3. Log in as CKB Dilsen-Stokkem
4. Submissions should be shown by default

## What to check

- Should we use `organization.naam` as default sort ?

# Links to other PR's

- https://github.com/lblod/frontend-worship-decisions/pull/8

# Notes

- This wont build because of readable-stream issue, a [maintenance](https://github.com/lblod/frontend-worship-decisions/pull/8) PR will be linked here and should be merged before this one.
